packages/workflow-core/src/lib/workflow-runner.ts

```diff
@@ -741,10 +768,20 @@ export class WorkflowRunner {
     }
 
     if (!!response) {
-      this.context.pluginsOutput = {
-        ...(this.context.pluginsOutput || {}),
-        ...{ [commonPlugin.name]: response },
-      };
+      if (hasPersistResponseDestination(commonPlugin)) {
+        if (response) {
```

Comment:
Seems like its redundant

Reply:
you receive either response or error.
we need to check if the response is appearing and not the error